docs/user_guide/5_data_management.md

```diff
@@ -97,4 +97,7 @@ media_client.upload("/Users/tom/Downloads/image.jpg", split="dev", object_detect
 
 # upload a folder of images and assign them to the 'train' split
 media_client.upload("/Users/tom/Downloads/images", split="train")
+
+# if you have created a metadata in the platform, you can also upload a value for that metadata
+media_client.upload("/Users/tom/Downloads/images/image1.png", metadata_dict={"cloudy": "true"})
```

Comment:
nit: might be helpful to document if the metadata key (e.g. "cloudy") is case sensitive.

Reply:
updated, thanks!